components/Footer.tsx

```diff
@@ -1,32 +1,64 @@
 import React from "react";
 import styles from "../styles/Footer.module.css";
+import Image from "next/image";
 
 export default function Footer() {
 	return (
 		<div className={styles.globalFooter}>
-			<div className={styles.medexLogo}>Medex</div>
 			<div className={styles.searchBoxLeft}>
-				<span className={styles.smallText}>
-					What are you looking for?
-				</span>
-				<input type="text" />
+				<h2>Medex</h2>
+				<p className={styles.smallText}>What are you looking for?</p>
+				<input
+					type="text"
+					name="search"
+					placeholder="Search.."
+					className={styles.inputSearch}
+				/>
 			</div>
 			<div className={styles.learnInfo}>
 				<h3>Learn</h3>
-				<p>About us</p>
-				<p>Our Experts</p>
-				<p>How it works</p>
-				<p>Purpose</p>
-				<p>Investors</p>
+				<p className={styles.linksUnderline}>About us</p>
+				<p className={styles.linksUnderline}>Our Experts</p>
+				<p className={styles.linksUnderline}>How it works</p>
+				<p className={styles.linksUnderline}>Purpose</p>
+				<p className={styles.linksUnderline}>Investors</p>
+				<p className={styles.linksUnderline}>Reviews</p>
 			</div>
-			<div className={styles.ContactInfo}>
+			<div className={styles.contactInfo}>
 				<h3>Contact</h3>
-				<p>Support</p>
-				<p>contact@medex.com</p>
+				<p className={styles.linksUnderline}>Support</p>
+				<p className={styles.linksUnderline}>contact@medex.com</p>
 				<div className={styles.smallIcons}>
-					<p>Instagram icon</p>
-					<p>Facebook icon</p>
-					<p>Twitter icon</p>
+					<div className={styles.oneIcon}>
+						<Image
+							src="/icons/instagram.png"
+							alt="Instagram"
+							width={36}
+							height={36}
+							layout="fixed"
+							className={styles.iconsImage}
+						/>
+					</div>
+					<div className={styles.oneIcon}>
+						<Image
+							src="/icons/facebook.png"
+							alt="Facebook"
+							width={36}
```

Comment:
rozmiar ikon powinien być stylowany css-em, coś ala `.oneIcon > img`
doceniam obecność alt-text

Reply:
https://github.com/arjamizo/medex-langing-page/pull/2/commits/16c66a0a02205327f6ff1bbe128dc281de87f362
szybki fix, zeby juz zaczac wersje PL, nie tak łatwo to zrobić w nextjs https://github.com/vercel/next.js/discussions/18739

Reply:
ok

Reply:
znalazłem fix dla komponentu Image
https://www.youtube.com/watch?v=P7i5YIJRJew
zaimplementuje za niedługo